app/build.gradle

```diff
@@ -26,18 +33,29 @@ android {
         }
     }
     compileOptions {
-        sourceCompatibility JavaVersion.VERSION_11
-        targetCompatibility JavaVersion.VERSION_11
+        sourceCompatibility JavaVersion.VERSION_1_8
```

Comment:
any reason to downgrade to 8?

Reply:
My intention was to remove one more step from the dev docs. But I didn't finish it (until just now). Now it's just using the defaults, and no extra configuration is needed, so I removed it.